### DISCUSSION TITLE
Why I can't use Image Optimization in SSG/export?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#25420] [Help]
I'm trying to use next image optimization, but I'm getting the below error...
Why I can't use Image Optimization in SSG/export?

![image](https://user-images.githubusercontent.com/20674439/119381578-9d4bb480-bc97-11eb-8659-be45e475d6af.png)

![image](https://user-images.githubusercontent.com/20674439/119381886-b3f20b80-bc97-11eb-8d1b-5122f436a56a.png)

Thanks so much!

**Comment by RohithChittibommala** [comment]
I am also facing same error, There is no much info on the web on how to optimize images on SSG next.js
